CMakeLists.txt

```diff
@@ -75,7 +75,14 @@ include(DefaultBuildType)
 #
 # Configuration
 #
+
+if(WIN32 AND (NOT CMAKE_INSTALL_SYSCONFDIR))
```

Comment:
This is not needed.  `CMAKE_INSTALL_SYSCONFDIR` is used only when you use the `install` command in CMake and have a `SYSCONF` parameter.

ccache is only installing the executable, no conf file is installed.

Reply:
This is interpolated into `config.h` [here](https://github.com/ccache/ccache/blob/master/cmake/config.h.in#L209).

If you build this branch and run `./ccache -vs` you will see that it takes effect.

Reply:
ok, but then it has a wrong value. `C:\ProgramData` is hardcoded. You can have Windows installed on a different drive, not just `C`. 

Running:

```
$ set | findstr ^ProgramData$
ALLUSERSPROFILE=C:\ProgramData
ProgramData=C:\ProgramData
```

I think `%allusersprofile%` should be used in this case.

Reply:
You're right, I forgot about this, I'll adjust the other gnuinstalldirs variables as well.

Reply:
By the way, have you tried working in PowerShell?

I am maintaining a guide for it [here](https://github.com/rkitover/windows-dev-guide).

Reply:
Can you take a look again please?

Reply:
Actually, now that I think about it again, the variable needs to be expanded at runtime, not compile time.

Reply:
Fixed.